tests/draft7/optional/format.json

```diff
@@ -248,6 +248,22 @@
             }
         ]
     },
+    {
+        "description": "validation of an internationalized e-mail addresses",
+        "schema": {"format": "idn-email"},
+        "tests": [
+            {
+                "description": "a valid idn e-mail address (example@example.test in Hangul)",
+                "data": "실례@실례.테스트",
```

Comment:
If you're gonna put the literal bytes here, might want to document that our test suite files themselves are UTF-8, not sure that's documented anywhere already.

Reply:
Should that just go in the README or is there something else to do for it?  I've always been a bit vague on that- it tends to Just Work(tm) on MacOS whenever I've needed it.